spec/helpers/for.js

```diff
@@ -10,6 +10,48 @@ describe('for helper', function() {
     // Build a test runner
     const runTestCases = testRunner({context});
 
+    it('should iterate once if zeroDiffStart is true.', function(done) {
+        runTestCases([
+            {
+                input: '{{#for 1 this zeroDiffStart=true}}{{$index}}:{{name}} {{/for}}',
+                output: '1:Joe ',
+            },
+            {
+                input: '{{#for 1 1 this zeroDiffStart=true}}{{$index}}:{{name}} {{/for}}',
+                output: '1:Joe ',
+            },
+            {
+                input: '{{#for 0 0 this zeroDiffStart=true}}{{$index}}:{{name}} {{/for}}',
+                output: '0:Joe ',
+            },
+            {
+                input: '{{#for 1000 1000 this zeroDiffStart=true}}{{$index}}:{{name}} {{/for}}',
+                output: '1000:Joe ',
+            },
```

Comment:
I'd add one more case of `for 1 2` just to make this even more clear that this applies to more than just a single iteration.